<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_room</class>
 <widget class="QWidget" name="create_room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>501</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="textBoxName">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter a name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
                          background-color: #4C9F70; 
                          color: white; 
                          font-size: 16px; 
                          border: 4px solid #2C6B43; 
                          border-radius: 5px; 
                          padding: 10px 10px;
                          text-align: center;
                          }
QPushButton:pressed {
                          background-color: #3E7F58;
}
QPushButton:hover {
                          background-color: #3E7F58;
}</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
                          background-color: #4C9F70; 
                          color: white; 
                          font-size: 16px; 
                          border: 4px solid #2C6B43; 
                          border-radius: 5px; 
                          padding: 10px 10px;
                          text-align: center;
                          }
QPushButton:pressed {
                          background-color: #3E7F58;
}
QPushButton:hover {
                          background-color: #3E7F58;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
